crates/cheatcodes/spec/src/vm.rs

```diff
@@ -325,6 +329,10 @@ interface Vm {
     #[cheatcode(group = Evm, safety = Unsafe)]
     function warp(uint256 newTimestamp) external;
 
+    /// Gets the current `block.timestamp`
+    #[cheatcode(group = Evm, safety = Unsafe)]
+    function getBlockTimestamp() external returns (uint256 timestamp);
```

Comment:
```suggestion
    #[cheatcode(group = Evm, safety = Safe)]
    function getBlockTimestamp() external view returns (uint256 timestamp);
```

Reply:
done in https://github.com/foundry-rs/foundry/pull/6630/commits/20e1e769aebdbe53d11d90059d7a6a883d27b4d2